BenchPress/Helpers/BenchPress.Azure/AksCluster.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+﻿using module ./public/classes/ConfirmResult.psm1
```

Comment:
Just wondering, was there a specific reason why this file and the test file name was changed?

Reply:
In my attempt to follow best practices for project folder structure, when I created my two new classes I put them in the `./public/classes` folder. I decided to go ahead and move this one at the same time.

Reply:
Sorry, don't know to comment on overall files in Github like I can with AzDO. I should be more specific. Was wondering why the `AKSCluster.ps1` and `AKSCluster.Tests.ps1` files were renamed to `AksCluster`? I do think it fits CamelCase better.

Reply:
Ah, sorry, completely missed the point of the question... yes, it was for best practices for camel casing. Plus, it reflects Microsoft's Az PowerShell cmdlet naming.

There are typically two schools of thought when it comes to Pascal and camel casing, and neither of them supports three capital letters in a row. One school of thought is that any acronym, regardless of length, should only ever have the initial character capitalized. The other school of thought is that acronyms that are two characters long can be capitalized.

Microsoft is in the latter camp: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers